content/guides/models/registry/lineage.md

```diff
@@ -0,0 +1,94 @@
+---
+description: Create a lineage map in the W&B Registry.
+menu:
+  default:
+    parent: registry
+    identifier: lineage
+title: Create and view lineage maps
+weight: 8
+---
+
+Within a collection in the W&B Registry, you can view a history of the artifacts that were created and used throughout an ML experiment. This history is called a _lineage graph_.
+
+{{% alert %}}
+You can also view lineage graphs for artifacts you log to W&B that are not part of a collection.
+{{% /alert %}}
+
+<!-- Lineage graphs can show the run that logged an artifact as well as artifacts used by a specific run. -->
+
+Lineage graphs can show the specific run that logs an artifact. In addition, lineage graphs can also show which run used an artifact as an input. In other words, lineage graphs can show the input and output of a run. 
+
+
+For example, the proceeding image shows artifacts created and used throughout an ML experiment:
+
+{{< img src="/images/registry/registry_lineage.png" alt="" >}}
+
+From left to right, the image shows.
+1. Multiple runs log the "split_zoo_dataset:v4" artifact.
+2. The "rural-feather-20" run uses the "split_zoo_dataset:v4" artifact for training.
+3. The output of the "rural-feather-20" run is a model artifact called "zoo-ylbchv20:v0".
+4. A run called "northern-lake-21" uses the model artifact "zoo-ylbchv20:v0" to evaluate the model.
+
+
+## Track the input of a run
+
+Declare an artifact as an input to a W&B run with the `wandb.init.use_artifact` API. The `wandb.init.use_artifact` API marks the artifact as a dependency of the run.
+
+The proceeding code snippet shows how to use the `use_artifact`. Ensure to replace values enclosed in angle brackets (`< >`) with your values:
+
+```python
+import wandb
+
+# Initialize a run
+run = wandb.init(project="<project>", entity="<entity>")
+
+# Get artifact, mark it as a dependency
+artifact = run.use_artifact(artifact_or_name="<name>", aliases="<alias>")
+```
+
+Once you have retrieved your artifact, you can use that artifact to (for example), evaluate the performance of a model. 
+
+
+## Track the output of a run
+
+
+Track the output of a run by logging an artifact with the `wandb.init.log` API. The `wandb.init.log` API logs the artifact as an output of the run.
+
+Use ([`wandb.init.log_artifact`]({{< relref "/ref/python/run.md#log_artifact" >}})) to declare an artifact as an output of a run.
```

Comment:
Is `log` an alias to `log_artifact` or are these two sentences slightly different?